lib/Driver.cpp

```diff
@@ -115,7 +115,7 @@ std::unique_ptr<Result> Driver::Run() {
 
       auto MPoints = Finder.findMutationPoints(Ctx, *(testee->getTesteeFunction()));
       if (MPoints.empty()) {
-        Logger::debug() << "\t\no mutation points, skipping.\n";
+        Logger::debug() << "\t\tno mutation points, skipping.\n";
```

Comment:
Instead of using tabs, you can tell the different loggers to indent with a given numer of spaces using for example `Logger::debug().indent(8) << "message"`